lib/mongoHandler.js

```diff
@@ -30,7 +30,7 @@ MongoStore._isRelationshipAttribute = function(attribute) {
 
 
 MongoStore._toMongoDocument = function(resource) {
-  var document = _.clone(resource, true);
```

Comment:
We can 'un-require' clone if its been swapped to omit and isnt being used elsewhere